pulp_rpm/tests/functional/api/test_consume_content.py

```diff
@@ -200,8 +201,10 @@ def setUpClass(cls):
         cls.pkg_mgr = cli.PackageManager(cls.cfg)
         cls.api_client = api.Client(cls.cfg, api.json_handler)
 
+        api_root = os.environ.get("PULP_API_ROOT", "/pulp/").lstrip("/")
+
         signing_services = cls.api_client.using_handler(api.page_handler).get(
-            "pulp/api/v3/signing-services/", params={"name": "sign-metadata"}
+            f"{api_root}api/v3/signing-services/", params={"name": "sign-metadata"}
```

Comment:
why would nor you user V3_API_ROOT from settings?

Reply:
because functional tests run outside of the container and dynaconf get these vars inside the container (when django app runs)